src/components/DpTreeList/DpTreeList.vue

```diff
@@ -18,6 +18,7 @@
           <slot name="header" />
         </div>
         <dp-tree-list-toggle
+          v-if="hasContent"
```

Comment:
shouldn't this be 
```suggestion
          v-if="treeData.length > 0"
```
? 
Or are there circumstances where we want to handle that independent?

Reply:
No, I didn't realize I could use this, Thx I changed it. 666140d25b132afd337e96297bb4348e81c521b2